core-ui/src/components/Predefined/Create/ResourceForm/ResourceForm.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
As I've mentioned before - I don't think this file should be in `Predefined/Create` - even though it's probably going to be used only by components in here.
I think it would make more sense to leave this directory for actual Resource definitions, and move the common library to `shared`